src/skmultiflow/data/random_tree_generator.py

```diff
@@ -6,56 +6,57 @@
 
 class RandomTreeGenerator(Stream):
     """ Random Tree stream generator.
-       
-    This generator is built based on its description in Domingo and Hulten's 
-    'Knowledge Discovery and Data Mining'. The generator is based on a random 
+
+    This generator is built based on its description in Domingo and Hulten's
+    'Knowledge Discovery and Data Mining'. The generator is based on a random
     tree that splits features at random and sets labels to its leafs.
-    
-    The tree structure is composed on Node objects, which can be either inner 
-    nodes or leaf nodes. The choice comes as a function fo the parameters 
+
+    The tree structure is composed on Node objects, which can be either inner
+    nodes or leaf nodes. The choice comes as a function fo the parameters
     passed to its initializer.
-    
-    Since the concepts are generated and classified according to a tree 
+
+    Since the concepts are generated and classified according to a tree
     structure, in theory, it should favour decision tree learners.
-    
+
     Parameters
     ----------
     tree_random_state: int (Default: None)
         Seed for random generation of tree.
-    
+
     sample_random_state: int (Default: None)
         Seed for random generation of instances.
-    
+
     n_classes: int (Default: 2)
         The number of classes to generate.
-    
+
     n_cat_features: int (Default: 5)
-        The number of categorical features to generate. Categorical features are binary encoded, the actual number of
-        categorical features is `n_cat_features`x`n_categories_per_cat_feature`
-    
+        The number of categorical features to generate. Categorical features are binary encoded,
+        the actual number of categorical features is
+        `n_cat_features`x`n_categories_per_cat_feature`
```

Comment:
Is this line break needed?

Reply:
Then the length of line 34 will be 100 characters.

Reply:
You are right :D